src/Resources/contao/config/config.php

```diff
@@ -77,13 +77,34 @@
         'callback' => "\WEM\SmartgearBundle\Backend\Smartgear",
     ],
 ]);
-
+array_insert(
+    $GLOBALS['BE_MOD']['content'],
+    array_search('article', array_keys($GLOBALS['BE_MOD']['content']), true) + 1,
+    [
+        'wem_sg_social_link' => [
+            'callback' => "\WEM\SmartgearBundle\Backend\SocialLink",
+        ],
+    ]);
+array_insert(
+    $GLOBALS['BE_MOD'],
+    array_search('content', array_keys($GLOBALS['BE_MOD']), true) + 1,
```

Comment:
Place this configuration into system section, we do not want the user to update the options.